src/Tizen.NUI.Components/Controls/Scrollbar.cs

```diff
@@ -214,30 +214,72 @@ public Extents TrackPadding
 
         /// <inheritdoc/>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public override void Initialize(float contentLength, Size viewSize, float currentPosition, bool isHorizontal = false)
+        public override void Initialize(float contentLength, float viewportLength, float currentPosition, bool isHorizontal = false)
```

Comment:
If possible,
it will be better if user can set this parameter through property.